client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -11,56 +11,54 @@ import FormButton from 'components/forms/form-button';
 import StepHeader from 'signup/step-header';
 import Button from 'components/button';
 import { abtest } from 'lib/abtest';
+import { localize } from 'i18n-calypso';
 
-export default React.createClass( {
-	displayName: 'BluehostStoreStep',
+export const BluehostStoreStep = props => {
+	const { translate } = props;
 
-	redirectToPartner() {
+	const redirectToPartner = () => {
 		if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
 			window.location.href = 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce';
 		} else {
 			window.location.href = 'https://www.bluehost.com/wordpress';
 		}
-	},
+	};
```

Comment:
It doesn't look like this function has anything particularly to do with the component, so why not shove it out above into the module space?